lib/convection/model/template/resource/aws_sqs_queue_policy.rb

```diff
@@ -8,9 +9,25 @@ class Resource
         # AWS::SQS::QueuePolicy
         ##
         class SQSQueuePolicy < Resource
+          extend Forwardable
+
           type 'AWS::SQS::QueuePolicy'
-          property :queue, 'Queues', :type => :list
-          property :policy_document, 'PolicyDocument'
+          property :queues, 'Queues'
```

Comment:
Yeah it is. Here is the CF json.
{
   "Type" : "AWS::SQS::QueuePolicy",
   "Properties" : {
      "PolicyDocument" : JSON,
      "Queues" : [ String, ... ]
   }
}

So just change this back to property :queue, 'Queues, :type => :list?